config/routes.rb

```diff
@@ -27,6 +27,8 @@
       get "/linked/:content_id", to: "link_sets#get_linked"
 
       get "/linkables", to: "content_items#linkables"
+
+      get "/grouped_content_and_links", to: "grouped_content_and_links#index"
```

Comment:
Yep. Actually we had trouble coming up with a descriptive name at all, so I'm interested if anyone has suggestions for something better to call this endpoint.

I don't think we have established language for talking about the thing that the content ID identifies - it's not something that's explicitly named in the data model at the moment. Other apps have a Document/Edition distinction but publishing api just has groups of Content Items that relate to each other.